console/EAP7-1159_Support_runtime_operations_for_messaging_servers.adoc

```diff
@@ -0,0 +1,100 @@
+= Support runtime operations for messaging servers
+:author:            Harald Pehl
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
+
+== Overview
+
+There are lots of new runtime operations for messaging servers, which are currently not supported in HAL. This RFE is about to add support for these operations.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-1159[EAP7-1159]
+
+=== Dev Contacts
+
+* mailto:hpehl@redhat.com[Harald Pehl]
+
+=== QE Contacts
+
+* mailto:pjelinek@redhat.com[Pavel Jelinek]
+
+=== Affected Projects or Components
+
+* HAL
+
+== Requirements
+
+The console adds the following actions to the messaging server runtime column:
+
+* View: Will open a new application screen (see below)
+* `export-journal()`
```

Comment:
this operation will create a XML file on the server file system (in the same directory where the journal log files are stored)